cmd/collection/main.go

```diff
@@ -147,6 +155,14 @@ func main() {
 		// epoch qc contract flags
 		flags.BoolVar(&insecureAccessAPI, "insecure-access-api", false, "required if insecure GRPC connection should be used")
 		flags.StringSliceVar(&accessNodeIDS, "access-node-ids", []string{}, fmt.Sprintf("array of access node IDs sorted in priority order where the first ID in this array will get the first connection attempt and each subsequent ID after serves as a fallback. Minimum length %d. Use '*' for all IDs in protocol state.", common.DefaultAccessNodeIDSMinimum))
+
+		// dynamic node startup flags
+		flags.BoolVar(&dynamicStartup, "dynamic-startup", false, "when set to true the node will attempt to wait for a specified epoch counter & phase before initializing")
```

Comment:
I actually think we can safely remove this flag in tandem with some other suggested changes. 

In the PreInit, we first check for a root snapshot. 
* If there is no snapshot, and `dynamic-join-access-pubkey` and `dynamic-join-access-address` are set, we can initiate Dynamic-Join. 
* If there is a snapshot, we skip Dynamic-Join start up using the snapshot.

Reply:
https://github.com/onflow/flow-go/pull/1893/commits/8ea656ac507e9969c635a600e908891394d7ac2c